packages/analytics-js-plugins/src/errorReporting/utils.ts

```diff
@@ -142,8 +143,25 @@ const getBugsnagErrorEvent = (
   ],
 });
 
+/**
+ * A function to determine whether the error should be promoted to notify or not
+ * @param {Error} error
+ * @returns
+ */
+const isAllowedToBeNotified = (errorMessage: string) =>
+  !ERROR_MESSAGES_TO_BE_FILTERED.some(e => errorMessage.includes(e));
+
+/**
+ * A function to determine if the error is from Rudder SDK
+ * @param {Error} event
+ * @returns
+ */
 const isRudderSDKError = (event: any) => {
   const errorOrigin = event.stacktrace?.[0]?.file;
+  const errorMessage = event.message;
+  if (errorMessage && typeof errorMessage === 'string' && !isAllowedToBeNotified(errorMessage)) {
```

Comment:
Do not combine error filtering action into this method. It has a different responsibility.